sources/3.0.0/sections/10-data_product_format.adoc

```diff
@@ -654,9 +666,70 @@ For example, "latitude,longitude". Reverse scan direction along an axis is indic
 
 |===
 
-==== Feature Instance group -- BathymetryCoverage.01
+[[note13]]
+[NOTE]
+====
+The number depends on the number of different vertical datum areas in the Feature Container Group in the Feature Container Group.
```

Comment:
`The number depends on the number of different vertical datum areas in the Feature Container Group in the Feature Container Group.`
Should probably be: The number depends on the number of different vertical datum areas in the Feature Container Group. (Without repetition)

Reply:
done in b7d5ead54bb2e04311fbd840167a0a3d19b89a21